src/services/db/GitFileSystemService.ts

```diff
@@ -333,6 +333,12 @@ export default class GitFileSystemService {
         )
 
         if (error instanceof GitError) {
+          if (
```

Comment:
would you know why the request even contains things like //filename?

are these actually broken links that cms tries to resolve?

Reply:
> would you know why the request even contains things like //filename?
we have a markdown editor, so there are possibility for typo + the logs suggest this has occurred at least once. 
we dont debounce too, to this could occur while the user is typing the correct string. 

> are these actually broken links that cms tries to resolve?
yes

Reply:
can we add comments for this check here so that we don't lose context of when this error statements occur?